commands/certmanager/delete.go

```diff
@@ -54,38 +53,34 @@ func CmdDelete(c *core.CommandConfig) error {
 		return err
 	}
 
+	certs, _, err := c.CertificateManagerServices.Certs().List()
```

Comment:
We only want to list the certificates if `--all` is set ,  so lets move this back under `if allFlag {`